src/v2/view-builder/views/EnrollProfileView.js

```diff
@@ -29,5 +38,32 @@ const Footer = BaseFooter.extend({
 
 export default BaseView.extend({
   Body,
-  Footer
+  Footer,
+  createModelClass (currentViewState, optionUiSchemaConfig, settings) {
+    let ModelClass = BaseView.prototype.createModelClass.apply(this, arguments);
+
+    settings.parseSchema(currentViewState.uiSchema,
+      (schema) => {
+        currentViewState.uiSchema = schema;
+        ModelClass = BaseView.prototype.createModelClass.apply(this, currentViewState, optionUiSchemaConfig);
```

Comment:
`.call(this, currentViewState, optionUiSchemaConfig, settings)`

Reply:
🤦